docs/dev/cut-a-new-release.md

```diff
@@ -204,3 +205,23 @@ If you need to release a hotfix for a previously released version (that is: not
     * This is important so that no changes are lost.
     * This might have merge conflicts with the `master` branch, which you need to fix before it is merged.
 8. Once the above pull request lands, delete the support branch.
+
+## Upgrading android-components to a new version of Glean
+
+On Android, Mozilla products consume the Glean SDK through its wrapper in [`android-components`](https://github.com/mozilla-mobile/android-components).
+Therefore, when a new Glean SDK release is made, `android-components` must also be updated.
+
+After following one of the above instructions to make a Glean SDK release:
+
+1) Ensure that CI has completed and the artifacts have published to [the Glean GitHub releases page](https://github.com/mozilla/glean/releases/).
```

Comment:
While this seems to render as a list just fine, should we use the name `<number>.` format as for other lists in this document, for consistency?